snapd-glib/snapd-client-sync.c

```diff
@@ -519,6 +519,36 @@ snapd_client_get_interfaces_sync (SnapdClient *client,
     return snapd_client_get_interfaces_finish (client, data.result, plugs, slots, error);
 }
 
+/**
+ * snapd_client_get_interface_info_sync:
+ * @client: a #SnapdClient.
+ * @names: (array zero-terminated=1): a null-terminated array of interface names or %NULL.
+ * @flags: control what information is returned about the interfaces.
```

Comment:
Use the same form as other flags:

`@flags: a set of #SnapdInterfaceFlags to control what information is returned about the interfaces.`

That way there is an easy link to the flags in the documentation.